source/fundamentals/enterprise-auth.txt

```diff
@@ -158,11 +158,21 @@ You may need to specify one or more of the following additional
       - :java-docs:`JAVA_SASL_CLIENT_PROPERTIES_KEY <apidocs/mongodb-driver-core/com/mongodb/MongoCredential.html#JAVA_SASL_CLIENT_PROPERTIES_KEY>`
       - :java-docs:`JAVA_SUBJECT_PROVIDER_KEY <apidocs/mongodb-driver-core/com/mongodb/MongoCredential.html#JAVA_SUBJECT_PROVIDER_KEY>`
 
-      Your code to instantiate a ``MongoClient`` using GSSAPI and additional
-      properties might look something like this:
+      Select the **SERVICE_NAME_KEY** or **JAVA_SUBJECT_KEY** tab to
```

Comment:
Use `:guilabel:` for names of UI elements like tabs.

Reply:
Good point, thanks for pointing this out. Going forward, I will try and remember to use the `:guilabel:` directive when referencing UI elements.